test/modellib/ConvModel.cpp

```diff
@@ -99,7 +99,7 @@ bool Conv2DLibBuilder::build() {
 
```

Comment:
Perhaps, you shoudl change `-1` in Line 63-64 to `ShapedType::kDynamic`too.

Lines 63-64:
```
  if (isDynamic)
    N1 = CIn1 = COut1 = H1 = W1 = -1;
```

Reply:
Thanks. I did do this yesterday in Conv and the other model builder files, but I'm still working on another issue.